test/mpi/maint/test_coll_algos.sh

```diff
@@ -141,8 +141,8 @@ for algo_name in ${algo_names}; do
                 env="${testing_env} env=MPIR_CVAR_IALLREDUCE_INTRA_ALGORITHM=${algo_name}"
                 env="${env} env=MPIR_CVAR_IALLREDUCE_RECEXCH_KVAL=${kval}"
 
-                echo "allred 4 arg=100 ${env}" >> ${testlist_cvar}
-                echo "allred 7 ${env}" >> ${testlist_cvar}
+                echo "allred 4 arg=-count=100 arg=-evenmemtype=host arg=-oddmemtype=host ${env}" >> ${testlist_cvar}
```

Comment:
We made it so the MTest macros default to host memory type if not specified, so these extra arguments are not needed.

Reply:
Ok. I missed that part. I will look into it again and fix these.

Reply:
> BTW, I tested this manually with both ch4:ofi and ch4:ucx and the new tests look to work fine.

That's great :)